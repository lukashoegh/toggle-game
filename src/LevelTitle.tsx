import * as React from 'react';

interface P {
    title: string;
    author: string;
}

class LevelTitle extends React.Component<P, null> {
    render() {
        return (
            <div>
                <h1>{this.props.title}</h1>
                <p>by {this.props.author}</p>
            </div>
        );
    }
}

export default LevelTitle