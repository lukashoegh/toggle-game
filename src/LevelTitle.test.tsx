import * as ReactTestRender from 'react-test-renderer';
import LevelTitle from './LevelTitle';
import * as React from 'react';

it('renders correctly', () => {
    const tree = ReactTestRender.create(
        <LevelTitle title="Test Title" author="Firstname McSecondname" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});