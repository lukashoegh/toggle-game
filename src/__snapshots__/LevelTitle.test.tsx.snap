exports[`test renders correctly 1`] = `
<div>
  <h1>
    Test Title
  </h1>
  <p>
    by 
    Firstname McSecondname
  </p>
</div>
`;
